cocotb/share/makefiles/simulators/Makefile.ius

```diff
@@ -62,6 +62,12 @@ else
     EXTRA_ARGS +=
 endif
 
+ifdef RTL_LIBRARY
```

Comment:
something like this would not be shorter?
```
RTL_LIBRARY ?= $(TOPLEVEL)
MAKE_LIB = -makelib $(RTL_LIBRARY )
```

Reply:
Oh sure, thanks for pointing this out.